memor/template.py

```diff
@@ -154,12 +155,26 @@ def from_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
                 loaded_obj = json_object.copy()
             content = loaded_obj["content"]
             title = loaded_obj["title"]
-            memor_version = loaded_obj["memor_version"]
             custom_map = loaded_obj["custom_map"]
+            memor_version = loaded_obj["memor_version"]
             date_created = datetime.datetime.strptime(loaded_obj["date_created"], DATE_TIME_FORMAT)
             date_modified = datetime.datetime.strptime(loaded_obj["date_modified"], DATE_TIME_FORMAT)
         except Exception:
             raise MemorValidationError(INVALID_TEMPLATE_STRUCTURE_MESSAGE)
+        _validate_string(content, "content")
+        if title:
+            _validate_string(title, "title")
+        _validate_custom_map(custom_map)
+        _validate_string(memor_version, "memor_version")
+        return content, title, memor_version, custom_map, date_created, date_modified
```

Comment:
The `PromptTemplate` class has many parameters, so returning them in this format is not a good approach. I suggest returning a dictionary instead.

Reply:
fixed in [9523075](https://github.com/openscilab/memor/pull/120/commits/9523075ef2ebd4a782024e43d9cd231834bd51a5)